mappings/net/minecraft/util/Identifier.mapping

```diff
@@ -19,7 +19,7 @@ CLASS net/minecraft/class_2960 net/minecraft/util/Identifier
 		ARG 0 id
 		ARG 1 delimiter
 	METHOD method_12831 isCharValid (C)Z
-		ARG 0 c
+		ARG 0 ch
```

Comment:
In other places in `Identifier`, `method_12834` and `method_12837` (maybe lambda methods), the parameter is `c`. Consider changing them to `ch` as well, or change this one back to `c`.
```suggestion
		ARG 0 c
```